recipe/meta.yaml

```diff
@@ -14,13 +14,14 @@ source:
 build:
   number: 0
   noarch: python
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: "python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
 
 requirements:
-  build:
+  host:
     - python
     - setuptools
     - pyscaffold <2.6
+    - pip
   run:
     - python
     - setuptools
```

Comment:
I don't think setuptools is needed at runtime?

Reply:
It is not needed anymore but in general this version cannot be released on conda-forge as long as `tsfresh` still needs such an old version of `pyscaffold`.